package-lock.json

```diff
@@ -4906,7 +4900,15 @@
       "integrity": "sha1-MAV0OOrGz3+MR2fzhkjWaX11yQM=",
       "dev": true,
       "requires": {
-        "minimist": "0.2.1"
+        "minimist": "0.0.8"
```

Comment:
did we accidentally downgrade `minimist`?

Reply:
no, i think this is due to my past PR not having done `npm ci`. Once this serialize issue is fixed i can go back and fix minimist if it pops up again.

Reply:
This will downgrade minimist to a vulnerable version, we should revert it.